<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>IP地址:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>to</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>scan</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="2" column="0" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>265</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>by https://github.com/whoAmIsiTerces</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
